src/setup/cortx_deploy

```diff
@@ -110,6 +111,46 @@ node_id=`cat /etc/machine-id`
 cluster_id=$(conf_get $(yaml_config $CLUSTER_INFO) "cluster>id")
 log $INFO "Starting provisioning on node $node_id which is part of cluster $cluster_id."
 
+deploy=false
+upgrade=false
+# Check current image version
+CONFSTOR_FILE=$(awk -F:// '{print $2}' <<< $CORTX_CONFSTORE)
+# check if confstore file is empty or not present
+if [ ! -s "$CONFSTOR_FILE" ]; then
+    log $INFO "$CORTX_CONFSTORE file not found or file is empty"
+    log $INFO "deployment set to true"
```

Comment:
This can be debug message...

Reply:
Removed this check as confstore is not to be considered as a file.